src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -16,36 +16,86 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
 ---
 
-This document describes some best practices and examples to help you get the most out of your alerts configuration.
+Improve your alert coverage by implementing the following recommendations. 
+
+Read on to learn the best practices for:
+- [Policies](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#policy-practices)
+- [Notification channels](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#channel-practices)
+- [Incident preferences](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#incident-practices)
+- [Thresholds and violations](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#threshold-practices)
+- [Muting roles](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#mute-practices)
 
 <Callout variant="important">
-  Before reading this, we recommend you read [Alerts concepts and workflow](/docs/alerts/new-relic-alerts/getting-started/alert-policy-workflow).
+  Before reading this, we recommend you also read [Alerts concepts and workflow](/docs/alerts/new-relic-alerts/getting-started/alert-policy-workflow).
```

Comment:
I'm super cold on this callout. It's kind of a drag to immediately send someone somewhere else :) Consider deleting!

If you want to keep it, and I think there are arguments for that, then I think it would be helpful to add some information about why we think someone should read that other doc.

Something like: "This doc describes how the different parts of alerts work together to notify you about problems in your system." or something

Reply:
Also, if we're sending someone somewhere else as prerequisite knowledge, what would it look like to fold that content into this doc? 

To be clear, I'm not asking you to literally do that :) More of a thought experiment.

Reply:
I agree--I didn't know how to keep or remove it. I'm fine deleting it.